### PR TITLE
Add topika

### PR DESCRIPTION
**Pull request by ltalirz**
* provided PR for including license file https://github.com/muhrin/topika/pull/2

### CODE REVIEW

**Comment by ltalirz** [comment]
Is this not the [recommended way](https://github.com/conda-forge/apispec-feedstock/pull/22#issuecomment-432191517) to add 'dev' dependencies?
```yaml
channel_sources:
  - conda-forge,conda-forge dev,defaults
```

**Comment by ltalirz** [comment]
Seems it doesn't work yet...
More discussion [here](https://github.com/conda-forge/staged-recipes/pull/6893)

**Comment by ltalirz** [comment]
@conda-forge/staged-recipes 

Same problem as in https://github.com/conda-forge/staged-recipes/pull/6893#issuecomment-435620403

CI is failing because of the inability to pick up a required package with a `dev` label, but I will fix the feedstock using `conda smithy rerender` once it is created.

Can the PR be merged?

**Comment by scopatz** [comment]
I am not sure that it is safe to merge a package that is failing like this because it might not render correctly, and thus block other PRs to staged-recipes from becoming feedstocks.

**Comment by ltalirz** [comment]
@scopatz Right, I can understand that. How would you then recommend to proceed?

**Comment by scopatz** [comment]
I would recommend getting the dependency that is missing into conda-forge so that this will then work.

**Comment by ltalirz** [comment]
Currently, fresh PRs to staged recipes can't depend on the dev channel, but I understand than one can fix this once one has a stand-alone feedstock (using `conda smithy rerender`).
From https://github.com/conda-forge/staged-recipes/pull/6893#issuecomment-434514551 I understood that the current suggested approach is to merge the PR and then fix the problem in the feedstock afterwards.

**Comment by scopatz** [comment]
Then it needs to get into the main label

**Comment by ltalirz** [comment]
> Then it needs to get into the main label

Just to make sure I understand correctly:
Are you suggesting that the 1.0.0b pika release should go into the main conda channel?

If that is the suggested policy, that's perfectly fine with me, it's just that the maintainer of the pika recipe got a different impression of how it should be done.

**Comment by scopatz** [comment]
Yes, that is the suggestion, and it is against our standard policy, but in this case (since it is blocking downstream packages) I think we can make an exception.

**Comment by ltalirz** [comment]
@scopatz Ok, thanks for the advice and the lightning fast replies!
I will contact the maintainer of the pika recipe then.

**Comment by ltalirz** [comment]
@scopatz Following your advice, I added the pika 1.0.0b1 release also in the `master` branch of the pika feedstock - https://github.com/conda-forge/pika-feedstock/pull/12

The build after the merge completed successfully but for some reason the 1.0.0b1 version does not show up in search results (and the topika build still fails)
```
$ conda search -c conda-forge pika
Loading channels: done
# Name                  Version           Build  Channel
pika                     0.10.0          py27_0  conda-forge
pika                     0.10.0          py34_0  conda-forge
pika                     0.10.0          py35_0  conda-forge
pika                     0.10.0          py36_0  conda-forge
pika                     0.11.0          py27_0  conda-forge
pika                     0.11.0          py35_0  conda-forge
pika                     0.11.0          py36_0  conda-forge
pika                     0.11.2          py27_0  conda-forge
pika                     0.11.2          py35_0  conda-forge
pika                     0.11.2          py36_0  conda-forge
pika                     0.12.0          py27_0  conda-forge
pika                     0.12.0          py35_0  conda-forge
pika                     0.12.0          py36_0  conda-forge
pika                     0.12.0            py_1  conda-forge
```

Just to show that the (unchanged) recipe for pika 1.0.0b1 worked just fine before in the `dev` channel:
```
$ conda search -c conda-forge/label/dev pika
Loading channels: done
# Name                  Version           Build  Channel
pika                    1.0.0b1            py_0  conda-forge/label/dev
```

I don't really know what else to do. Any ideas?

**Comment by ltalirz** [comment]
@jjhelmus Thanks for the clarification!
I.e. is adding the "main" label to pika 1.0.0b1 something I can do? Is it something a maintainer can do?

**Comment by jjhelmus** [comment]
I believe only core conda-forge members can change labels on Anaconda.org.  Not because we don't trust maintainers but rather because of the lack of per package permissions on Anaconda.org.  It looks like the "main" label has been added to pika 1.0.0b1.

**Comment by ltalirz** [comment]
Ah thanks - I guess that was @scopatz . 
Build passing now.

**Comment by scopatz** [comment]
Yep that was me :)

File: recipes/topika/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+test:
+  imports:
+    - topika
+
+about:
+  home: https://github.com/muhrin/topika.git
+  license: Apache
+  license_family: APACHE
+  license_file: ''
```
[the remark this answers is not in the document]
Comment (ltalirz) [reply recipes/topika/meta.yaml]: thanks, wasn't aware of this.
I did already open a PR to include the license and [it is merged](https://github.com/muhrin/topika/pull/2), so we can remove this again in the next conda release

File: recipes/topika/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  summary: A python remote communications library
+
+extra:
+  recipe-maintainers: 
+  - ltalirz
+  - muhrin
```
[the remark this answers is not in the document]
Comment (muhrin) [reply recipes/topika/meta.yaml]: Dear @scopatz , yes, no problem.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**